components/dashboard/src/settings/Integrations.tsx

```diff
@@ -43,6 +43,7 @@ function GitProviders() {
     const [disconnectModal, setDisconnectModal] = useState<{ provider: AuthProviderInfo } | undefined>(undefined);
```

Comment:
**question**: Couldn't find a better line to add this comment, but now that we have this blocking alert in place, what do you think of also skip removing the disconnect action from the more actions dropdown and show the same alert when a user tried to disconnect the last remaining provider?

| Two connected providers | One connected provider |
|-|-|
| <img width="769" alt="disconnect-2" src="https://user-images.githubusercontent.com/120486/152786198-f9d86a61-c597-4ee7-a707-e95a65b0a13f.png"> | <img width="769" alt="disconnect-1" src="https://user-images.githubusercontent.com/120486/152786205-a30d62a1-e65d-4f3b-96d4-87cc786c6408.png"> |